src/shared/main/java/com/couchbase/lite/FileLogger.java

```diff
@@ -50,10 +50,7 @@ public void log(@NonNull LogLevel level, @NonNull LogDomain domain, @NonNull Str
 
     @NonNull
     @Override
-    public LogLevel getLevel() {
-        logLevel = Log.getLogLevelForC4Level(C4Log.getBinaryFileLevel());
-        return logLevel;
-    }
+    public LogLevel getLevel() { return logLevel; }
```

Comment:
I just noticed that default log level is set to warning in the reset() method. The default log level of the FileLogging should be none.

Reference Spec: https://docs.google.com/document/d/1V_kVZuvckAPX41obYsu9SAqRuLxwe0TLbZDrMA_InEE

Reply:
Why?

Reply:
FileLogging is disabled by default so the LogLevel is none.

Reply:
Fixed.  Default LogLevel is now NONE.  See line 132.